benchmarks/src/main/java/org/apache/druid/benchmark/query/ScanBenchmark.java

```diff
@@ -107,12 +107,15 @@
   @Param({"200000"})
   private int rowsPerSegment;
 
-  @Param({"basic.A"})
+  @Param({"basic.A", "basic.B", "basic.C", "basic.D"})
   private String schemaAndQuery;
 
   @Param({"1000", "99999"})
   private int limit;
 
+  @Param({"none", "descending", "ascending"})
+  private static String timeOrdering;
```

Comment:
The basicA(), basicB()... query builder functions are static (like in SearchBenchmark).  Since timeOrdering is used when building the query, it also needs to be static.